dotnet/CoreLib/AppBuilders/HostedHandlersBuilder.cs

```diff
@@ -17,19 +16,19 @@ public static HostApplicationBuilder CreateApplicationBuilder(string[]? args = n
 
         if (Environment.GetEnvironmentVariable("ASPNETCORE_ENVIRONMENT")?.ToUpperInvariant() == "DEVELOPMENT")
         {
-            builder.Configuration.AddJsonFile("appsettings.Development.json", optional: true);
+            builder.Configuration.AddJsonFile("appsettings.development.json", optional: true);
```

Comment:
.NET usually has PascalCase values for "Development" and "Production" in the env var ASPNETCORE_ENVIRONMENT and I think this could be an issue in some case sensitive filesystems. I prefer lowercase too, but I'm not sure what's .NET expectation across OSes.

Reply:
Yeah, that occurred to me as an afterthought also...I'll rollback.